app/controllers/collection_templates_controller.rb

```diff
@@ -46,6 +52,21 @@ def create
     end
   end
 
+  def update
+    if names_list(params[:id]).include?(params[:draft][:template_name])
+      flash[:error] = 'A template with that name already exists.'
+      redirect_to manage_collections_path
+      return
```

Comment:
if you need to add a return after a redirect, I really like `redirect_to <some path> and return`

Reply:
I don't have a strong opinion on it, but I believe that rubocop style guide bans the use of 'and' and 'or'.

Reply:
yeah, but this is a specific pattern for rails, which is why I think it's ok and we have made an exception for it